### PR TITLE
Add run_exports and fix for cling/clang

### PR DESCRIPTION
**Pull request by saraedum**
@isuruf, hope you don't mind that I fix two things in one PR.

### CODE REVIEW

**Comment by saraedum** [comment]
@conda-forge-admin, please rerender

**Comment by saraedum** [comment]
@conda-forge-admin, please rerender

File: recipe/meta.yaml [hunk 1]
```diff
@@ -9,10 +9,16 @@ source:
...
+    - cling.patch
 
 build:
-  number: 1002
+  number: 1003
   skip: true  # [win]
+  run_exports:
+    # pin exact version of ppl, there has not been a release in many years so
+    # we have no idea what the next version is going to look like
+    - ppl
```
Comment (isuruf) [inline recipe/meta.yaml]: You need,
```
{{ pin_subpackage("ppl", max_pin="x.x") }}
``
Otherwise it doesn't pin a version.

Comment (saraedum) [reply recipe/meta.yaml]: I see. Had misread the documentation.

**Comment by saraedum** [comment]
@isuruf: I think this is ready to go :)

**Pull request merged**